deep-security-agent/meta/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+dependencies: []
+galaxy_info:
+  description: Install, Activate and Manage Trend Micro Deep Security Agent
+  min_ansible_version: 2.4
+  company: Trend Micro
+  author: Deep Security
+  license: MIT
+  galaxy_tags:
+    - trend-micro
+    - deep-security
+    - dsm
+    - dsa
+  platforms:
+    - name: Amazon
```

Comment:
For Amazon linux, I believe we support all versions? I think we can mark here as "N/A"

Reply:
Changed to "N/A" since Amazon Linux doesn't have specific release version.